realsense2_camera/scripts/rs2_listener.py

```diff
@@ -19,18 +19,12 @@
 from rclpy import qos
 from sensor_msgs.msg import Image as msg_Image
 import numpy as np
-import inspect
 import ctypes
 import struct
 import quaternion
-import os
-if (os.getenv('ROS_DISTRO') != "dashing"):
-    import tf2_ros
-if (os.getenv('ROS_DISTRO') == "humble"):
-    from sensor_msgs.msg import PointCloud2 as msg_PointCloud2
-    from sensor_msgs_py import point_cloud2 as pc2
-# from sensor_msgs.msg import PointCloud2 as msg_PointCloud2
-# import sensor_msgs.point_cloud2 as pc2
+import tf2_ros
+from sensor_msgs.msg import PointCloud2 as msg_PointCloud2
+from sensor_msgs_py import point_cloud2 as pc2
```

Comment:
This used to be `humble` only, is it relevant to `iron` and `rolling` too?

Reply:
did not understand the question.. can you elaborate more ? are you talking about pc2 or tf2_ros ?

Reply:
NVM I understand now :)